web-client/src/presenter/computeds/utilities/addConsolidatedProperties.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * This can add consolidatedToolTip, inConsolidatedGroup flag and/or inLeadCase flag to passed in object item.
+ *
+ * @param object
+ * @returns object
+ */
+
+export const addConsolidatedProperties = object => {
```

Comment:
Looks great, thanks for doing this! Would love to see some tests around it too but PR approved :D

Reply:
we added some tests 🥳

Reply:
Awesome! Thanks